Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/delete.go

```diff
@@ -73,6 +82,39 @@ func (f *Factory) NewCmdDelete(out io.Writer) *cobra.Command {
 	return cmd
 }
 
+func resourceByLabel(f *Factory, label string) (string, error) {
+	cmdNamespace, err := f.DefaultNamespace()
+	if err != nil {
+		return "", err
+	}
+	kc, err := f.Client()
+	if err != nil {
+		return "", err
+	}
+	selector, err := labels.Parse(label)
+	if err != nil {
+		return "", err
+	}
+	list, err := kc.Events(cmdNamespace).List(selector, fields.Everything())
```

Comment:
This currently doesn't work, but even if I specify `labels.Everything` instead of `selector` the only resources I get back are pods and nodes. Why's that? @smarterclayton @bparees

ps. I am running sample-app and https://github.com/openshift/ruby-hello-world so there are lots of resources ie. lots of events I should get back, right?